doc/reference/storage/settings/settings.rst

```diff
@@ -70,6 +70,23 @@ After all data is loaded, the ``h_commit`` handler is issued,
 signalling the application that the settings were successfully
 retrieved.
 
+Storing data to persistent storage
+**********************************
+
+A call to ``settings_save_one()`` and ``settings_save()`` uses an ``h_export``
```

Comment:
AFAIK only **settings_save** uses the module's h_export. **settings_save_one** have both key and value passed as arguments.

Reply:
Right - corrected.